docs/cli/reference.md

```
@@ -638,6 +650,136 @@ Move session to group
 
 
 
+## `aoe plugin`
+
+Manage plugins (install, enable, disable, update)
+
+**Usage:** `aoe plugin <COMMAND>`
+
+###### **Subcommands:**
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h6

(MD001, heading-increment)